model/bin/cmplr.env

```diff
@@ -147,7 +147,8 @@ if [ "$cmplr" == "intel" ]          || [ "$cmplr" == "intel_debug" ]    || [ "$c
     optc="$optc -xHOST"
     optl="$optl -xHOST"
   elif [ ! -z "$(echo $cmplr | grep wcoss2)" ] ; then
-    optc="$optc -fp-model precise -ftz -fast-transcendentals -no-prec-div -no-prec-sqrt -align array64byte"
+    # optc="$optc -fp-model precise -ftz -fast-transcendentals -no-prec-div -no-prec-sqrt -align array64byte"
```

Comment:
Is there any reason to keep this line commented out?

Reply:
I want to keep the original one for the future reference. However, I am OK to delete this statement if you think it is not needed.